beacon-chain/rpc/prysm/v1alpha1/beacon/attestations.go

```diff
@@ -67,45 +69,107 @@ func (bs *Server) ListAttestations(
 			return nil, status.Errorf(codes.Internal, "Could not fetch attestations: %v", err)
 		}
 	case *ethpb.ListAttestationsRequest_Epoch:
+		if q.Epoch >= params.BeaconConfig().ElectraForkEpoch {
```

Comment:
this seems kinda hacky, is this a change in the spec?

Reply:
this is gRPC, not Beacon API